.changeset/small-roses-fly.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
```

Comment:
Don't need to bump the `example-app` as it's not a published package.

Reply:
updated